app/main-quadratic-probing.py

```diff
@@ -0,0 +1,192 @@
+from typing import Union, Any, Iterable, Tuple
+
+
+# Quadratic Probing
+class Dictionary:
+    def __init__(self,
+                 initial_capacity: int = 8,
+                 load_factor: float = 0.75) -> None:
+        # Cells will store (key, value) or None
+        self.table: list[Tuple[Any, Any]] = \
+            [None] * initial_capacity
+        self.size: int = 0
+        self.load_factor = load_factor
+        self.capacity = initial_capacity
+
+    def _hash(self, key: Any) -> int:
+        return hash() % self.capacity
```

Comment:
The `_hash` method is missing the key parameter in the `hash()` function call. This will cause a `TypeError` because `hash()` requires an argument. Ensure you pass the `key` to the `hash()` function.